src/clients/consumer/options.ts

```diff
@@ -8,6 +8,8 @@ export const groupOptionsProperties = {
   sessionTimeout: { type: 'number', minimum: 0 },
   rebalanceTimeout: { type: 'number', minimum: 0 },
   heartbeatInterval: { type: 'number', minimum: 0 },
+  groupProtocol: { type: 'string', enum: ['classic', 'consumer'] },
```

Comment:
Can you please define this values in `enumerations.ts`?

Reply:
done